application/server/main/tasks.py

```diff
@@ -112,81 +113,99 @@ def create_task_unpaywall(args):
         logger_console.debug(db_handler.fetch_all())
 
 
-def get_softcite_version(local_files):
-    softcite_files = [file for file in local_files if file.endswith('.software.json')]
-    if softcite_files:
-        with open(softcite_files[0], 'r') as f:
-            softcite_version = json.load(f)['version']
-    else:
-        softcite_version = "0"
+def get_softcite_version(softcite_file: str) -> str:
+    """Get the version of softcite used by reading from an output file"""
+    with open(softcite_file, 'r') as f:
+        softcite_version = json.load(f)['version']
```

Comment:
Maybe we could user `.get('version')` in case the is none.

Reply:
Pas utile à mon avis : ça fait partie du contrat d'interface avec softcite. Le cas où on appelle la fonction get_softcite_version implique nécessairement qu'on a un fichier de sortie de softcite. Du coup, si l'output de softcite  change, c'est un breaking change, on ne veut probablement pas que ça fail silencieusement, ce qui serait le cas avec un get.